README.md

```diff
@@ -0,0 +1,60 @@
+# UMfile_utils
+
+UMfile_util is a Python library that provides specific tools to edit UM dump and field files.
+
+# UMfile_utils Desciption
+
+This package will include three abilities
+
+### perturbIC.py 
+If the simulation fails, this code allows for minor random perturbation of the input theta (surface termperature) in the restart files. 
+Then the simulation can be rerun to avoid crashing. In the future functionality will be added to perturb other fields as well. 
+
+### um_fields_subset.py
+
+This code runs on a Dump file and allows the user to select a group of fields in the file to store 
+in a new fields file or exclude a group of fields from the file. Both of these functionalities will 
+create a new output file. 
+NEED TO ADD USE CASES
+
+### change_date.py
+NEED TO ADD DESCRIPTION 
```

Comment:
Add in that it allows the user to change the date and time that a simulation starts. Meant to operate on restart files?